server/views/pages/trivyScan.njk

```diff
@@ -65,7 +65,16 @@
           <td>{{ row.severity }}</td>
           <td><a href="{{ row.PrimaryURL }}">{{ row.vulnerabilityID }}</a></td>
           <td>{{ row.pkgName }}</td>
-          <td>{{ row.description }}</td>
+          <td>
```

Comment:
You should be using the actual components in the templates https://design-system.service.gov.uk/components/details/ so:

```{{ govukDetails({
  summaryText: "Help with nationality",
  text: "We need to know your nationality so we can work out which elections you’re entitled to vote in. If you cannot provide your nationality, you’ll have to send copies of identity documents through the post."
}) }}```

and making sure

```{% from "govuk/components/details/macro.njk" import govukDetails %}```

is at the top